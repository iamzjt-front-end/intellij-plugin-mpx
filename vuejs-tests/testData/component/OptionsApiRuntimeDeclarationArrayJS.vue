<script>
export default {
  name: "TestComponent",
  props: ["optional", "required"]
}
</script>

<template>
  {{optional}}
  {{required}}
</template>
