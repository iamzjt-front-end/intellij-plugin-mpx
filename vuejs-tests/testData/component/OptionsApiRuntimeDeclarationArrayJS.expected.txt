{
 class: "VueSourceComponent",
 defaultName: "TestComponent",
 props: [
  {
   class: "VueSourceInputProperty",
   name: "optional",
   required: false,
  },
  {
   class: "VueSourceInputProperty",
   name: "required",
   required: false,
  },
 ],
}
